x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -55,11 +55,14 @@ type AlertType = NonNullableProps<
   typeof ALERT_RULE_TYPE_ID | typeof ALERT_RULE_CONSUMER | typeof SPACE_IDS
 >;
 
-const isValidAlert = (source?: ParsedTechnicalFields): source is AlertType => {
+const isValidAlert = (source?: any): source is AlertType => {
```

Comment:
could we create a new type here instead of using `any`?

Reply:
I think you can use the type `SignalSourceHit` which uses `estypes.SearchHit`;

Reply:
I had to update the AlertType adding `{ _index: string; _id: string } & ...` in order to make the type guard work, but yes, it worked 👍